client.py

```diff
@@ -158,10 +158,10 @@ def recvall(buffer):  # function to receive large amounts of data
 
 # vbs message box
 def MessageBox(message):

```

Comment:
This seems to crash the client currently.

Reply:
Did you get an error message? I think it's because of the subprocess pipes i added, not sure though